openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST2/procedures/numpy.py

```diff
@@ -0,0 +1,11 @@
+# The standard way to import NumPy:
+import numpy as np
+
+# Create a 2-D array, set every second element in
+# some rows and find max per row:
+x = np.arange(15, dtype=np.int64).reshape(3, 5)
+x[1:, ::2] = -99
+print(x)
+# array([[  0,   1,   2,   3,   4],
+#        [-99,   6, -99,   8, -99],
+#        [-99,  11, -99,  13, -99]])
```

Comment:
I thought this script was a good addition to the demo and it lets us ensure the Python package installation process is working